tests/test_series.py

```diff
@@ -1236,3 +1236,24 @@ def cond2(x: pd.Series[int]) -> pd.Series[bool]:
 
     cond3 = pd.Series([False, True, True])
     check(assert_type(s.where(cond3, other=0), "pd.Series[int]"), pd.Series, int)
+
+
+def test_bitwise_operators_int() -> None:
```

Comment:
change name to `test_bitwise_operators()`

Reply:
done